image/cli/mascli/functions/gitops_db2u_database

```diff
@@ -272,6 +278,24 @@ function gitops_db2u_database_noninteractive() {
       --db2-timezone)
         export DB2_TIMEZONE=$1 && shift
         ;;
+      --db2-backup-bucket-name)
```

Comment:
Can we add a comment just above this line that says something like:
`# Optional DB2 Backup details`